variorum_power.json

```diff
@@ -0,0 +1,36 @@
+{"options":
+[
+{
+    "name" : "power_node_watts",
+    "category": "metric",
+    "description": "Node power (via variorum)",
+    "services": [ "variorum" ],
+    "config": 
+    {
+        "CALI_VARIORUM_DOMAINS": "power_node_watts"
+    },
+    "query":
+    [
+        { "level"  : "local", 
+          "let"    :
+          [ 
+            "power.avg=first(avg#variorum.val.power_node_watts,variorum.val.power_node_watts)",
+            "power.max=first(max#variorum.val.power_node_watts,variorum.val.power_node_watts)"
+          ],
+          "select" : 
+          [
+            { "expr": "max(power.max)", "as": "Power/node (max)", "unit": "Watts" },
+            { "expr": "avg(power.avg)", "as": "Power/node (avg)", "unit": "Watts" }
+          ]
+        },
+        { "level"  : "cross",
+          "select" :
+          [
+            { "expr": "max(max#power.max)", "as": "Power/node (max)", "unit": "Watts" },
+            { "expr": "avg(avg#power.avg)", "as": "Power/node (avg)", "unit": "Watts" }
+          ]
+        }
+    ]
+}
+]
+}
```

Comment:
We can remove this JSON file from the PR as we are still waiting on the variorum spack package.

Reply:
done